electron/gog/setup.ts

```diff
@@ -101,15 +144,21 @@ async function setup(appName: string): Promise<void> {
           } /versionName="${
             gameInfo.install.version
           }" /nodesktopshorctut /nodesktopshortcut`
-          const workingDir = actionArguments?.workingDir?.replace(
-            '%SUPPORT%',
-            `"${path.join(gameInfo.install.install_path, 'support', appName)}"`
-          )
-          const executablePath = path.join(
+
+          const supportPath = `"${path.join(
             gameInfo.install.install_path,
             'support',
-            appName,
-            executableName
+            appName
+          )}"`
+          const workingDir = actionArguments?.workingDir
```

Comment:
maybe checking if `workingDir` exists before continuing would be safer since I wonder if this `replace` will break the code if we get `undefined` here.